Code/Framework/AzFramework/Platform/Linux/AzFramework/Process/ProcessCommunicator_Linux.cpp

```diff
@@ -11,7 +11,32 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <sys/ioctl.h>
+#include <sys/epoll.h>
+
 #include <AzFramework/Process/ProcessCommunicator.h>
+#include <AzCore/Memory/SystemAllocator.h>
+
+namespace ProcessCommunicatorLinuxInternal
+{
+
+    class CommunicatorDataImpl : public AzFramework::StdInOutProcessCommunicatorData
+    {
+        public:
```

Comment:
nitpick: public is normally unindented inside the class
```suggestion
    class CommunicatorDataImpl : public AzFramework::StdInOutProcessCommunicatorData
    {
    public:
```